python/pyspark/ml/tuning.py

```diff
@@ -289,8 +392,60 @@ def copy(self, extra=None):
             extra = dict()
         return CrossValidatorModel(self.bestModel.copy(extra))
 
+    @since("2.0.0")
+    def write(self):
+        """Returns an JavaMLWriter instance for this ML instance."""
```

Comment:
"JavaMLWriter"  --> "MLWriter"  (This is an error in the pipeline.py docs too)  Please update this elsewhere in this file as well.